src/app/caesar-portal/profile/profile.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="icon" [ngClass]="{invisible: photoVisibility}" (click)="toggle()">
+    <img [src]="currentUser.image || defaultPhoto" alt="photo">
+</div>
+<aside class="sidebar" [ngClass]="{invisible: popupVisibility}" (mouseleave)="toggle()">
```

Comment:
it is a bit weird for me that you get to see details on click and returns back to photo on mouseleave. Is it like this in the story?